include/seqan3/io/sequence_file/input.hpp

```diff
@@ -330,10 +325,13 @@ class sequence_file_input
     using stream_char_type      = char;
     //!\}
 
-    /*!\brief The subset of seqan3::field IDs that are valid for this file; order corresponds to the types in
+#pragma GCC diagnostic push // avoid deprecation warning of field::seq_qual
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+   /*!\brief The subset of seqan3::field IDs that are valid for this file; order corresponds to the types in
      * \ref field_types.
      */
     using field_ids            = fields<field::seq, field::id, field::qual, field::seq_qual>;
```

Comment:
```suggestion
    using field_ids            = fields<field::seq, field::id, field::qual, field::_seq_qal_deprecated>;
```

We can now also use without adding more `#pragma GCC diagnostic push`

Reply:
Yes, that is much prettier! :+1: